cockroachcloud/cloud-sso.md

```diff
@@ -70,3 +70,11 @@ To remove a user's access to {{ site.data.products.db }} without deleting their
 ### Can admins require a particular login method for their {{ site.data.products.db }} organization?
 
 Yes, as long as Enterprise Authentication is enabled for your {{ site.data.products.db }} organization.
+
+### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
+
+Enterprise Authentication currently supports only a Service Provider (CockroachDB Cloud) initiated flow. The identity provider initiated flow can not be enabled yet.
+
+### What default role is assigned to users when autoprovisioning is enabled in a CockroachDB Cloud org?
```

Comment:
```suggestion
### What default role is assigned to users when auto-provisioning is enabled in a {{ site.data.products.db }} organization?
```
Is the feature. called "autoprovisioning" (one word? I've seen it with a hyphen so far. If it's really the former, we need to add it to the linter and make sure we are consistent. We should use the variables for Cloud and Enterprise consistently.